src/TorchSharp/NN/Sequential.cs

```diff
@@ -176,6 +181,41 @@ public override nn.Module apply(Action<nn.Module> fn)
                 }
             }
 
+            /// <summary>
+            /// Module indexer.
+            /// </summary>
+            [IndexerName("SequentialItems")]
+            public Sequential this[(int? start, int? end) index] {
+                get {
+                    var result = new Sequential(Array.Empty<torch.nn.Module<Tensor, Tensor>>());
+                    var start = index.start.HasValue ? index.start.Value : 0;
+                    var end = index.end.HasValue ? index.end.Value : _modules.Count;
+
+                    for (var i = start; i < _modules.Count && i < end; i++) {
+                        result.Add(_names[i], _modules[i]);
+                    }
+                    return result;
+                }
+            }
+
+#if !NETSTANDARD2_0_OR_GREATER
```

Comment:
I guess this condition should be 

`#if NETSTANDARD2_1_OR_GREATER`

Reply:
Hmm... We've been using the 2_0 for a while now (you introduced it, I believe). What's changed?

Reply:
And, by the way, it didn't work in the test project -- I had to use 'NETFRAMEWORK' instead.

Reply:
Looking more I think `#if  NETCOREAPP3_1_OR_GREATER` would be better.

>  We've been using the 2_0 for a while now (you introduced it, I believe). What's changed?

Nothing changed. original condition `!NETSTANDARD2_0_OR_GREATER` can cause issue in the future if start support like ns2.1 which exclude these new methods.

Reply:
So, then I should replace it elsewhere, too, then?

Reply:
You may address this in another PR. I think changing it would be good in general.